LICENSE

```diff
@@ -0,0 +1,201 @@
+      except as required for reasonable and customary use in describing the
+      origin of the Work and reproducing the content of the NOTICE file.
+
+   7. Disclaimer of Warranty. Unless required by applicable law or
+      agreed to in writing, Licensor provides the Work (and each
+      Contributor provides its Contributions) on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+      implied, including, without limitation, any warranties or conditions
+      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+      PARTICULAR PURPOSE. You are solely responsible for determining the
+      appropriateness of using or redistributing the Work and assume any
+      risks associated with Your exercise of permissions under this License.
+
+   8. Limitation of Liability. In no event and under no legal theory,
+      whether in tort (including negligence), contract, or otherwise,
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2017 Ericsson AB.
```

Comment:
From the summit some comments where about Ericsson feeling too much as owner of the code, so signing off with Ericsson AB may feel a bit wrong.

This may of coure need to be brought up on internal meetings, but google cloud platform is a good example where they sign off with the "comunity owners".
https://github.com/GoogleCloudPlatform/community/blob/master/LICENSE.md

Maybe something we could do as well?

Also should 2017 not be 2019?

Reply:
With regards to Google Cloud Platform the "Google Cloud Platform Community Authors" link actually points a file (AUTHORS) which has Google Inc. in it. I have to little hands on experience with this kind of stuff, but in the google case it seems more to show the original authors of the code and copyright holders. Can a community hold copyright?

Reply:
I copied this file from eiffel repo and the company name was left there during the copy. This is the default Apache license file and company name should not be there as we are not doing with the license itself. I'll create a new issue in the backlog to remove the company name from Apache license files in repositories.

in general, if we are adding new files (java code for example), then headers in those files contain company or individual name if the file is newly created. if an existing file is edited and depending on the type of the contribution, additional individual or company names can be appended to the existing copyright line. this should go into how to contribute documentation which i'll create an issue for.